### ISSUE TITLE
version 3.27 not working

### ISSUE THREAD

**Issue by zdm** [mrallen1/Pod-Perldoc#28]
Under linux:

```
$ perldoc POD::Perldoc
Can't find any loadable formatter class in Pod::Perldoc::Toterm Pod::Perldoc::Toterm Pod::Perldoc::ToTerm Pod::Perldoc::ToTERM Pod::Simple::term Pod::Simple::term Pod::Simple::Term Pod::Simple::TERM Pod::term Pod::term Pod::Term Pod::TERM Pod::Perldoc::Totext Pod::Perldoc::Totext Pod::Perldoc::ToText Pod::Perldoc::ToTEXT Pod::Simple::text Pod::Simple::text Pod::Simple::Text Pod::Simple::TEXT Pod::text Pod::text Pod::Text Pod::TEXT Pod::Perldoc::ToPod?!
Aborting
 at /usr/local/perlbrew/perls/perl-5.24.0/bin/perldoc line 6.
```

Under Windows

```
perldoc POD::Perldoc
Invalid parameter - -R
```

What "-R" parameter? I don't specify them at all.

**Comment by mrallen1** [comment]
Can I get some more information about these failures? How did you install perldoc? Using cpanm or a CPAN client? From an RPM by the distro? By hand?

**Comment by zdm** [comment]
Hi.
I am installed using cpanm on both systems (linux and windows).

**Comment by zdm** [comment]
Is it possible to revert latest changes?
I have seeing, that other users also complains about this.

**Comment by zdm** [comment]
Pls, look at line 1941 in Pod::Perldoc
```
local $ENV{MORE} = defined $ENV{MORE} ? "$ENV{MORE} -R" : "-R";
```
Are you sure, that -R is needed here?
more utility has no -R option.

**Comment by jkeenan** [comment]
See also https://rt.perl.org/Ticket/Display.html?id=130759, where this problem causes a failure in the working of ```perldoc``` in Strawberry Perl on Windows.  At the very least, the assignment to ```local $ENV{MORE}``` on FreeBSD should have been wrapped in a test for the operating system.

**Comment by mrallen1** [comment]
Thanks for the report. I have released 3.27_02 to CPAN with a fix for these issues. Please try that release.

**Comment by itcharlie** [comment]
Hello @mrallen1,
I updated rt#130759 with a comment that shows that Pod::Perldoc version 3.27_02 works in Windows with Strawberry Perl.

**Comment by mrallen1** [comment]
Thanks @itcharlie

**Comment by itcharlie** [comment]
Hello @mrallen1 ,

FYI rt#130759 was update by sawyerx, he is requesting to have a new stable version of Pod::Perldoc released with this fix in order to make this fix available in Perl 5.26.0 as well.

Best and thank you for your time.

**Comment by mrallen1** [comment]
I released 3.28 to CPAN with a fix for this issue so I'm going to close this.

**Comment by jkeenan** [comment]
Pod-Perldoc version 3.28 has been merged into Perl 5 blead.
```
$ git show
commit 6aff4bf3cdbe7898ab6d2fb6c691e278cfd5e80f
Author: James E Keenan <jkeenan@cpan.org>
Date:   Fri Mar 17 19:10:50 2017 -0400

    Sync Pod-Perldoc with CPAN version 3.28.
```
Thanks all for your attention to this issue.
